src/rubintv/models.py

```diff
@@ -43,7 +43,7 @@ def parse_filename(self, delimiter: str = "_") -> Tuple:
         date = nList[2]
         seq_str = nList[4][:-4]  # Strip extension
         if seq_str == "final":
-            seq = 0
+            seq = 99999
```

Comment:
I always worry about sentinel values that are valid data values.  I.e. there could possibly be a sequence of 100000 images and then this value would be meaningless.  I'm not sure if there is anything to do about it, but it's something to think about.

Reply:
That is indeed why we changed it from zero. 99,999 is more than 1 image per second for a whole day, and given we have a 2s readout this is considered a safe value (I think middleware only allocates enough bits for 99999 anyway, so it's actually impossible to go over this)